libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -24,19 +24,20 @@ export function toFragment(htmlStrings, ...values) {
   return fragment;
 }
 
-// TODO this is just prototyped
 export const getFedsPlaceholderConfig = () => {
-  const { locale, miloLibs, env } = getConfig();
+  const { locale } = getConfig();
   let libOrigin = 'https://milo.adobe.com';
+
   if (window.location.origin.includes('localhost')) {
     libOrigin = `${window.location.origin}`;
   }
 
-  if (window.location.origin.includes('.hlx.')) {
-    const baseMiloUrl = env.name === 'prod'
-      ? 'https://main--milo--adobecom.hlx.live'
-      : 'https://main--milo--adobecom.hlx.page';
-    libOrigin = miloLibs || `${baseMiloUrl}`;
+  if (window.location.origin.includes('.hlx.page')) {
+    libOrigin = 'https://main--milo--adobecom.hlx.page';
```

Comment:
I'm a bit confused since the PR title mentions `consumer` but we're essentially hardcoding `https://main--milo--adobecom.hlx.*` as the source for placeholders. Is it correct to assume that gnav placeholders are only managed on milo? Theoretically, every consumer can have its own `placeholders.json`, e.g. https://main--bacom--adobecom.hlx.page/placeholders.json

Reply:
Yes, we specifically want to hardcode the path for the placeholders and always fetch them from 
`milo.adobe.com/placeholders.json` or `https://main--milo--adobe.com.hlx.page/placeholders.json` for every consumer. 

There have been plenty of discussions about generally centralising vs. not - so far we have a hybrid model. 
Content can be anywhere, but placeholders will live within `sharepoint/milo/placeholders.json` and the language variants.

We only use the placeholders for very generic stuff, such as "Sign in" or a "Search" placeholder. There's 0 reason to burden every consumer to create and maintain that file, also it could lead to a lot of inconsistency across consumers.

Reply:
Another thing to add: we're using a separate sheet for the navigation placeholders, named 'feds'. This is because we want to keep things consistent, as Okan mentioned. We want to avoid cases where CC uses 'Sign In', but DC uses 'Log in'. It would also be a burden for authors to create and constantly sync these labels. And it will also be problematic for localisation.